code/modules/cm_aliens/XenoStructures.dm

```diff
@@ -96,9 +99,21 @@
 /obj/effect/alien/resin/attackby(obj/item/W, mob/user)
 	if(!(W.flags_item & NOBLUDGEON))
 		var/damage = W.force
-		if(W.w_class < 4 || !W.sharp || W.force < 20) //only big strong sharp weapon are adequate
-			damage /= 4
-		health -= damage
+		var/multiplier = 1
+		if(W.damtype == "burn") //Burn damage deals extra vs resin structures (mostly welders).
+			multiplier += 1
+			if(istype(W, /obj/item/tool/pickaxe/plasmacutter))
```

Comment:
What if it is burn type of damage but not a plasma cutter?

Reply:
I figure all burn should have some benefit; welding torches get kicked up to 30